packages/ods-react/src/components/combobox/src/components/combobox-label/ComboboxLabel.tsx

```diff
@@ -0,0 +1,29 @@
+import { Combobox as VendorCombobox } from '@ark-ui/react/combobox';
+import classNames from 'classnames';
+import { type ComponentPropsWithRef, type FC, type JSX, forwardRef } from 'react';
+import style from './comboboxLabel.module.scss';
+
+interface ComboboxLabelProp extends ComponentPropsWithRef<'label'> {
+}
+
+const ComboboxLabel: FC<ComboboxLabelProp> = forwardRef(({
```

Comment:
we've removed all internal label, component should all use the FormFieldLabel and update their code accordingly (we can check together changes that are implied)